modules/hooks.nix

```diff
@@ -2848,9 +2856,9 @@ in
       terraform-format =
         {
           name = "terraform-format";
-          description = "Format terraform (`.tf`) files.";
-          package = tools.terraform-fmt;
-          entry = "${hooks.terraform-format.package}/bin/terraform-fmt";
+          description = "Format Terraform (`.tf`) files.";
+          package = tools.terraform;
+          entry = "${hooks.terraform-format.package}/bin/terraform fmt -check -diff";
```

Comment:
I realized that the `terraform-fmt` script was not necessary. I tried calling directly  `terraform fmt -check -diff` from the hook `entry` and it worked fine. I created a test [repository](https://github.com/sestrella/test-pre-commit-hooks.nix/blob/main/devenv.yaml) to validate this. Here is the outcome of the `devenv test` command running on the CI:

<img width="1351" alt="image" src="https://github.com/cachix/git-hooks.nix/assets/2049686/b1aa994f-292c-454a-85ce-662dce8ef21f">